### PR TITLE
LPS-131914 Avoid make a request if the dataDefinitionId or dataLayoutId do not exists

### PR DESCRIPTION
**Pull request by interaminense**

### CODE REVIEW

**Review by miltonmc — CHANGES_REQUESTED** [review]
Well done!

Just a minor comment

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/hooks/useData.js [hunk 1]
```diff
@@ -93,23 +87,35 @@ const DEFAULT_DATA_LAYOUT = {
...
-		fetchRetry: {
-			attempts: 0,
-		},
-		link: `${window.location.origin}/o/data-engine/v2.0/data-layouts/${dataLayoutId}`,
-	});
+	let dataDefinition = DEFAULT_DATA_DEFINITION;
+	let dataLayout = DEFAULT_DATA_LAYOUT;
+	const doFetch = useResource;
+
+	if (Number(dataDefinitionId)) {
```
[the remark this answers is not in the document]
Comment (interaminense) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/hooks/useData.js]: Done, see it here https://github.com/liferay-forms/liferay-portal/pull/224/commits/55bef920c82fe04d35dd2bbf8ab0fb59f5f361a4

**Review by miltonmc — APPROVED** [review]

**Comment by miltonmc** [comment]
ci:forward

**Pull request closed without merging**